test.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash -eu
 
-./build/checkops &> /dev/null
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
 
+# Failed tests can leave dangling mounts behind.
 for i in $(mount | grep -e "/tmp/encfs-reverse-tests-\|/tmp/encfs-tests-" | cut -f3 -d" "); do
 	echo "Warning: unmounting leftover filesystem: $i"
-	fusermount -u $i
+	fusermount -u $i || true
```

Comment:
An error here should not keep the test suite from running. We may get `device or resource busy` or even `command not found` (on MacOS). Tests should still be run.